cypress/support/index.d.ts

```diff
@@ -1,9 +1 @@
 import '@mate-academy/cypress-tools/support';
-
-declare global {
-  namespace Cypress {
-    interface Chainable<Subject> {
-      byDataCy(name: string, text: string): Chainable<JQuery>;
-    }
-  }
-}
\ No newline at end of file
```

Comment:
There should be a newline at the end of the file according to standard coding conventions.